modules/conversion-center/widgets/link-in-bio.php

```diff
@@ -0,0 +1,1200 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Widgets;
+
+use Elementor\Controls_Manager;
+use Elementor\Group_Control_Background;
+use Elementor\Modules\ConversionCenter\Classes\Render\Core_Render;
+use Elementor\Modules\ConversionCenter\Module as ConversionCenterModule;
+use Elementor\Plugin;
+use Elementor\Widget_Base;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+/**
+ * Elementor Link in Bio widget.
+ *
+ * Elementor widget that displays an image, a bio, up to 4 CTA links and up to 5 icons.
+ *
+ * @since 3.23.0
+ */
+class Link_In_Bio extends Widget_Base {
+
+	public function get_name(): string {
+		return 'link-in-bio';
+	}
+
+	public function get_title(): string {
+		return esc_html__( 'Link In Bio', 'elementor' );
+	}
+
+	public function get_icon(): string {
+		return 'eicon-bullet-list';
+	}
+
+	public function get_categories(): array {
+		return [ 'general' ];
+	}
+
+	public function get_keywords(): array {
+		return [ 'buttons', 'bio', 'widget' ];
+	}
+
+	public function show_in_panel(): bool {
+		return Plugin::$instance->experiments->is_feature_active( ConversionCenterModule::EXPERIMENT_NAME );
+	}
+
+	public function get_stack( $with_common_controls = true ): array {
+		return parent::get_stack( false );
+	}
+
+	protected function register_controls(): void {
+		$this->add_identity_section();
+
+		$this->end_controls_section();
+
+		$this->add_bio_section();
+
+		$this->end_controls_section();
+
+		$this->add_icons_controls();
+
+		$this->add_cta_controls();
+
+		$this->add_style_tab();
+
+		$this->add_advanced_tab();
+	}
+
+	protected function render(): void {
+		$render_strategy = new Core_Render();
+
+		$render_strategy->render( $this );
+	}
+
+	private function add_cta_controls(): void {
+		$this->start_controls_section(
+			'cta_section',
+			[
+				'label' => esc_html__( 'CTA Links', 'elementor' ),
+				'tab'   => Controls_Manager::TAB_CONTENT,
+			]
+		);
+
+		$this->add_control(
+			'cta_section_alert',
+			[
+				'type'       => Controls_Manager::ALERT,
+				'alert_type' => 'info',
+				'content'    => wp_kses_post(
+					__( 'Add up to <b>4</b> CTA links', 'elementor' )
+				),
+			]
+		);
+
+		$this->add_control(
+			'cta_link',
+			[
+				'type'          => Controls_Manager::REPEATER,
+				'fields'        => [
+					[
+						'name'        => 'cta_link_text',
+						'label'       => esc_html__( 'Text', 'elementor' ),
+						'type'        => Controls_Manager::TEXT,
+						'dynamic'     => [
+							'active' => true,
```

Comment:
Please use the Repeater class to generate the controls:

```
$repeater = new Repeater();

$repeater->add_control(
	'title',
	[
		'type' => Controls_Manager::TEXT,
		'label_block' => true,
		'required' => true,
	]
);
```

Reply:
Done